1.20/Dockerfile.fedora

```diff
@@ -59,19 +59,17 @@ COPY ./root/ /
 # writable as OpenShift default security model is to run the container under
 # random UID.
 RUN sed -i -f ${NGINX_APP_ROOT}/nginxconf-fed.sed ${NGINX_CONF_PATH} && \
-    chmod a+rwx ${NGINX_CONF_PATH} && \
     mkdir -p ${NGINX_APP_ROOT}/etc/nginx.d/ && \
     mkdir -p ${NGINX_APP_ROOT}/etc/nginx.default.d/ && \
     mkdir -p ${NGINX_APP_ROOT}/src/nginx-start/ && \
     mkdir -p ${NGINX_CONTAINER_SCRIPTS_PATH}/nginx-start && \
     mkdir -p ${NGINX_LOG_PATH} && \
     mkdir -p ${NGINX_PERL_MODULE_PATH} && \
+    chmod a+rwx ${NGINX_CONF_PATH} && \
     chmod -R a+rwx ${NGINX_APP_ROOT}/etc && \
-    chmod -R a+rwx /var /run && \
+    chmod -R a+rwx ${NGINX_APP_ROOT}/src/nginx-start/  && \
```

Comment:
What about to have 
```suggestion
    chmod -R a+rwx ${NGINX_APP_ROOT}/src && \
```

Reply:
That should also be OK. But I wanted to go with minimal changes to the permissions.